src/sire/CMakeLists.txt

```diff
@@ -4,8 +4,8 @@
 #
 ################################
 
-cmake_minimum_required(VERSION 3.3.0 FATAL_ERROR)
-cmake_policy(VERSION 3.3)
+cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
+cmake_policy(VERSION 3.5)
```

Comment:
Is CMake 3.5 needed? We've recently updated to 3.3.0 and above, which required updates and fixes to many of the CMake files. If this isn't strictly necessary, could you leave as before. (We plan on updating further at some point so that we are future proofed.)